src/routes/index.astro

```diff
@@ -2,4 +2,4 @@
 
 ---
 
-<meta http-equiv="refresh" content="0;url=/en/" />
+<h1>No ssg yet</h1>
```

Comment:
Do we have an idea what to do with this page?

Reply:
It's just there to catch the routing for static site generation.

Reply:
(which we don't have yet)